Resources/public/js/views/services/ez-locationviewviewservice.js

```diff
@@ -43,6 +44,150 @@ YUI.add('ez-locationviewviewservice', function (Y) {
         },
 
         /**
+         * _sendContentToTrashConfirmBox event handler on send to trash button
+         *
+         * @method _sendContentToTrashConfirmBox
+         * @protected
+         * @param {Object} e event facade of the sendToTrashAction event
+         */
+        _sendContentToTrashConfirmBox: function (e) {
+            e.preventDefault();
+            this.fire('confirmBoxOpen', {
+                config: {
+                    title: "Are you sure you want to send this content to trash?",
+                    confirmHandler: Y.bind(function () {
+                        this._sendToTrash(e);
```

Comment:
instead of passing the event facade, I would pass what you really need (the content AFAICS) so that this method can maybe be reused else where.